### ISSUE TITLE
Instructions about forking needs small clarification

### ISSUE THREAD

**Issue by talllguy** [github/government.github.com#215]
Small clarification request to the instruction on the [community page](http://government.github.com/community/). As @nschonni pointed out in a [pull request comment](https://github.com/github/government.github.com/pull/213#issuecomment-29498464), I accidentally created my branch off of the accessibility branch. I recommend the instructions include a bit that the gh-pages branch should be selected first.

**Comment by benbalter** [comment]
@talllguy late to respond, but thanks for opening the issue. The instructions have since been updated, so I don't believe this is a problem any more.
